src/const_eval.md

```diff
@@ -53,7 +54,7 @@ to be run.
 A _const context_ is one of the following:
 
 * [Array type length expressions]
-* Repeat expression length expressions
+* [Array repeat expressions][array expressions]
```

Comment:
Specifically only the length field of the repeat expression

Reply:
Oops, sorry!  I probably shouldn't open PRs late at night.